daemon/daemon.go

```diff
@@ -728,6 +724,10 @@ func NewDaemon(dp datapath.Datapath, iptablesManager rulesManager) (*Daemon, *en
 	)
 	policymap.InitMapInfo(option.Config.PolicyMapMaxEntries)
 
+	if err := bpf.ConfigureResourceLimits(); err != nil {
```

Comment:
Unit tests on Travis seem to fail because of that:

https://travis-ci.com/cilium/cilium/builds/124444793

Maybe we should configure memory limits only when we don't run with DryMode? Like

```
if !option.Config.DryMode {
	if err := bpf.ConfigureResourceLimits(); err != nil {
		log.WithError(err).Fatal("Unable to set memory resource limits")
	}
}
```